src/styles/partials/components/_site-nav.scss

```diff
@@ -0,0 +1,162 @@
+$desktop-menu-width: 400px;
+$mobile-menu-width: 245px;
+
+@mixin nav-menu-width($width) {
+  #bc_site-nav .bc_site-nav__list-container {
+    width: $width;
+
+    &.invisible {
+      transform: translateX(-$width);
+    }
+  }
+
+  .is-active {
+    .bc_site-nav__toggle-button {
+      transform: translateX($width) !important;
+    }
+  }
+
+  #bc_page.is-disabled {
+    transform: translateX($width);
+  }
+}
+
+#bc_site-nav {
+  position: relative;
+  top: 0;
+
+  .bc_site-nav__toggle-button {
+    position: absolute;
+    top: 10px; /* mimic the top border of the site */
+    transform: translateX(0);
+    z-index: $max-z-index;
+    @include default-transition(transform);
+  }
+
+  .bc_site-nav__seal {
+    max-width: 75px;
+    width: 100%;
+  }
+
+  .bc_site-nav__list-container {
+    background: $pale-blue-dark;
+    border-top: $border-width-largest solid $red;
+    color: $white;
+    padding: $default-padding $padding-large;
+    position: fixed;
+    transform: translateX(0);
+    overflow-y: auto;
+    top: 0;
+    bottom: 0;
+    @include default-transition(transform);
+    z-index: $max-z-index-minus-1;
+
+    ul {
+      list-style-type: none;
+      margin: $margin-large 0 0;
+      padding: 0;
+
+      a {
+        border: $border-width-small solid $white;
+        color: $white;
+        display: block;
+        font-family: $default-heading-font;
+        font-weight: $font-bold;
+        font-size: $small-heading-font-size;
+        margin: $default-margin auto;
+        padding: $padding-large $default-padding;
+        position: relative;
+        text-decoration: none;
+        text-transform: uppercase;
+
+        i {
+          text-align: center;
+          margin-right: $margin-small;
+          width: 40px;
```

Comment:
I think this is similar to some other icon centering, maybe it can be a mixin. @theilman7 I think you worked on something like this, do you remember where it was?